.github/changelog.yml

```diff
@@ -0,0 +1,3 @@
+filename: CHANGELOG.md
+include:
+  - /^bin\/|^config\/|^dev-site-config\/|^scripts\/|^src\/|^terra-favicon\/|^tests\/|^translations\/|^package.json/
```

Comment:
i don't think you have a config or bin folder or some others, probably doesn't hurt anything but you could shorten this list.

Reply:
I'll shorten it up